Tasks/Common/MSBuildHelpers/PathFunctions.ps1

```diff
@@ -193,16 +206,16 @@ function Get-VisualStudio_15_0 {
                             $null = $output.AppendLine($_)
                         }
                     }
-                $script:visualStudioCache['15.0'] = (ConvertFrom-Json -InputObject $output.ToString()) |
+                $script:visualStudioCache["$MajorVersion.0"] = (ConvertFrom-Json -InputObject $output.ToString()) |
```

Comment:
not sure whether you can just use `visualStudioCache[$MajorVersion]` as key, do we have to do `"$MajorVersion.0"`?